package.json

```diff
@@ -43,7 +45,8 @@
     "rollup-plugin-terser": "^7.0.2"
   },
   "scripts": {
-    "build": "rollup -c build-settings/rollup.config.js",
+    "build": "NODE_ENV=production rollup -c build-settings/rollup.config.js",
+    "watch": "NODE_ENV=development rollup -c build-settings/rollup.config.js --watch",
```

Comment:
Added a `watch` option as I think it will be useful - such as if I'm using Quokka to test the actual package output and want to tweak things as I go.